testing/infra/terraform/modules/ec_deployment/scripts/drop_pipeline.tftpl

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -eo
+
+ingest_pipelines=("logs-apm.app@custom" "logs-apm.error@custom" "metrics-apm.app@custom" "metrics-apm.internal@custom" "traces-apm.rum@custom" "traces-apm@custom")
```

Comment:
This is fine for now, but may be a little brittle and friction-inducing as we introduce more data streams. I've had this thought in my mind for a little while, and finally have written an issue to implement it: https://github.com/elastic/apm-server/issues/9339. If/when we do, we should come back and update this to make use of it.